### PR TITLE
K8SPS-152 - Add expose option for orchestrator in cr.yaml

### PR DESCRIPTION
**Pull request by tplavcic**
[K8SPS-152](https://jira.percona.com/browse/K8SPS-152) [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=percona&utm_campaign=badge_info)

ClusterIP is used by default, but option is needed if you want to set it to LoadBalancer

### CODE REVIEW

File: deploy/cr.yaml [hunk 1]
```diff
@@ -123,6 +123,9 @@ spec:
 #                - e2e-az1
 #                - e2e-az2
 
+    expose:
```
Comment (hors) [inline deploy/cr.yaml:126]: please comment it by default

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| auto-config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/da39040/auto-config.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/da39040/demand-backup.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/da39040/init-deploy.log) |
| gr-init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/da39040/gr-init-deploy.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/da39040/monitoring.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/da39040/scaling.log) |
| semi-sync | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/da39040/semi-sync.log) |
| config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/da39040/config.log) |
| service-per-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/da39040/service-per-pod.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/da39040/users.log) |
| sidecars | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/da39040/sidecars.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/da39040/limits.log) |
| We run 12 out of 12|

commit: https://github.com/percona/percona-server-mysql-operator/pull/165/commits/da39040b1688cf7f20179c76ad0ac14414c01833
image: `perconalab/percona-server-mysql-operator:PR-165-da39040`

File: deploy/cr.yaml (hunk 1 above)
Comment (tplavcic) [reply deploy/cr.yaml:126]: Should we comment it if by default it is exposed via ClusterIP (there is no enable/disable option)?

Comment (tplavcic) [reply deploy/cr.yaml]: I have commented it out for the moment, because all others are commented as well.

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| auto-config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/47d527c/auto-config.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/47d527c/init-deploy.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/47d527c/demand-backup.log) |
| gr-init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/47d527c/gr-init-deploy.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/47d527c/scaling.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/47d527c/monitoring.log) |
| semi-sync | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/47d527c/semi-sync.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/47d527c/users.log) |
| config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/47d527c/config.log) |
| service-per-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/47d527c/service-per-pod.log) |
| sidecars | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/47d527c/sidecars.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/47d527c/limits.log) |
| We run 12 out of 12|

commit: https://github.com/percona/percona-server-mysql-operator/pull/165/commits/47d527ce36111315822e872f3a57bf2612b5e7b3
image: `perconalab/percona-server-mysql-operator:PR-165-47d527c`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| auto-config | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/8df7237/auto-config.log) |
| config | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/8df7237/config.log) |
| demand-backup | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/8df7237/demand-backup.log) |
| scaling | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/8df7237/scaling.log) |
| users | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/8df7237/users.log) |
| init-deploy | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/8df7237/init-deploy.log) |
| gr-init-deploy | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/8df7237/gr-init-deploy.log) |
| monitoring | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/8df7237/monitoring.log) |
| semi-sync | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/8df7237/semi-sync.log) |
| service-per-pod | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/8df7237/service-per-pod.log) |
| sidecars | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/8df7237/sidecars.log) |
| limits | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/8df7237/limits.log) |
| We run 12 out of 12|

commit: https://github.com/percona/percona-server-mysql-operator/pull/165/commits/8df7237b407900b7f87e91f48477e6476c4ba0fb
image: `perconalab/percona-server-mysql-operator:PR-165-8df7237`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| auto-config | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/8df7237/auto-config.log) |
| config | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/8df7237/config.log) |
| init-deploy | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/8df7237/init-deploy.log) |
| gr-init-deploy | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/8df7237/gr-init-deploy.log) |
| monitoring | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/8df7237/monitoring.log) |
| semi-sync | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/8df7237/semi-sync.log) |
| service-per-pod | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/8df7237/service-per-pod.log) |
| sidecars | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/8df7237/sidecars.log) |
| limits | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/8df7237/limits.log) |
| demand-backup | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/8df7237/demand-backup.log) |
| scaling | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/8df7237/scaling.log) |
| users | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/8df7237/users.log) |
| We run 12 out of 12|

commit: https://github.com/percona/percona-server-mysql-operator/pull/165/commits/8df7237b407900b7f87e91f48477e6476c4ba0fb
image: `perconalab/percona-server-mysql-operator:PR-165-8df7237`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| auto-config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/93bc01e/auto-config.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/93bc01e/demand-backup.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/93bc01e/init-deploy.log) |
| gr-init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/93bc01e/gr-init-deploy.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/93bc01e/monitoring.log) |
| semi-sync | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/93bc01e/semi-sync.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/93bc01e/scaling.log) |
| service-per-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/93bc01e/service-per-pod.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/93bc01e/users.log) |
| sidecars | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/93bc01e/sidecars.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/93bc01e/limits.log) |
| config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/93bc01e/config.log) |
| We run 12 out of 12|

commit: https://github.com/percona/percona-server-mysql-operator/pull/165/commits/93bc01eee7128dc56b40e61811e406d4632b57b7
image: `perconalab/percona-server-mysql-operator:PR-165-93bc01e`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| auto-config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/224a690/auto-config.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/224a690/demand-backup.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/224a690/init-deploy.log) |
| gr-init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/224a690/gr-init-deploy.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/224a690/monitoring.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/224a690/scaling.log) |
| config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/224a690/config.log) |
| semi-sync | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/224a690/semi-sync.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/224a690/users.log) |
| service-per-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/224a690/service-per-pod.log) |
| sidecars | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/224a690/sidecars.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/224a690/limits.log) |
| We run 12 out of 12|

commit: https://github.com/percona/percona-server-mysql-operator/pull/165/commits/224a690fb7f7366e14a080d89fc087ab277998a0
image: `perconalab/percona-server-mysql-operator:PR-165-224a690`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| auto-config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/439dc67/auto-config.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/439dc67/demand-backup.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/439dc67/init-deploy.log) |
| gr-init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/439dc67/gr-init-deploy.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/439dc67/monitoring.log) |
| semi-sync | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/439dc67/semi-sync.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/439dc67/scaling.log) |
| config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/439dc67/config.log) |
| service-per-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/439dc67/service-per-pod.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/439dc67/users.log) |
| sidecars | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/439dc67/sidecars.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/439dc67/limits.log) |
| We run 12 out of 12|

commit: https://github.com/percona/percona-server-mysql-operator/pull/165/commits/439dc67cc01bb772177f9a439249edb6ac48f8a3
image: `perconalab/percona-server-mysql-operator:PR-165-439dc67`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| auto-config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/5171711/auto-config.log) |
| demand-backup | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/5171711/demand-backup.log) |
| config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/5171711/config.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/5171711/init-deploy.log) |
| gr-init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/5171711/gr-init-deploy.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/5171711/monitoring.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/5171711/scaling.log) |
| semi-sync | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/5171711/semi-sync.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/5171711/users.log) |
| service-per-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/5171711/service-per-pod.log) |
| sidecars | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/5171711/sidecars.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/5171711/limits.log) |
| We run 12 out of 12|

commit: https://github.com/percona/percona-server-mysql-operator/pull/165/commits/5171711758e2da0b8e7b434bfce7ee6ef5c091c9
image: `perconalab/percona-server-mysql-operator:PR-165-5171711`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| auto-config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/5171711/auto-config.log) |
| demand-backup | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/5171711/demand-backup.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/5171711/init-deploy.log) |
| gr-init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/5171711/gr-init-deploy.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/5171711/monitoring.log) |
| semi-sync | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/5171711/semi-sync.log) |
| config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/5171711/config.log) |
| service-per-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/5171711/service-per-pod.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/5171711/scaling.log) |
| sidecars | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/5171711/sidecars.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/5171711/limits.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/5171711/users.log) |
| We run 12 out of 12|

commit: https://github.com/percona/percona-server-mysql-operator/pull/165/commits/5171711758e2da0b8e7b434bfce7ee6ef5c091c9
image: `perconalab/percona-server-mysql-operator:PR-165-5171711`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| auto-config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/b03907d/auto-config.log) |
| demand-backup | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/b03907d/demand-backup.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/b03907d/init-deploy.log) |
| gr-init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/b03907d/gr-init-deploy.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/b03907d/monitoring.log) |
| semi-sync | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/b03907d/semi-sync.log) |
| service-per-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/b03907d/service-per-pod.log) |
| config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/b03907d/config.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/b03907d/scaling.log) |
| sidecars | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/b03907d/sidecars.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/b03907d/limits.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-165/b03907d/users.log) |
| We run 12 out of 12|

commit: https://github.com/percona/percona-server-mysql-operator/pull/165/commits/b03907d3039ef3618eba0d41405e60d2ba1868cf
image: `perconalab/percona-server-mysql-operator:PR-165-b03907d`

**Comment by tplavcic** [comment]
@hors Please check this and I propose we merge it so that other PR's pick this up and test results become relevant again.

**Review by hors — APPROVED** [review]

**Pull request merged**